leaf-ui/js/object/evoObjects.js

```diff
@@ -441,7 +406,7 @@ class EVO {
      * @param {*} intentionEval four digit code that corresponds to evidence pair (ex. 0011)
      */
     static getColor(intentionEval) {
-        if(IntentionColoring.isColorBlindMode) {
+        if(this.isColorBlindMode) {
```

Comment:
Add space between `if` and `(this.isColorBlindMode)`
Also, try to change it to `EVO.isColorBlindMode` for consistency